### PR TITLE
T-14700 - E2E tests

### PR DESCRIPTION
**Pull request by commoddity**
Adds a full E2E testing suite to the CI run.

### CODE REVIEW

File: tests/e2e_test.go [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+	t.NoError(err)
+	t.True(blockchainActivated)
+
+	activatedBlockchain, err := get[repository.Blockchain](fmt.Sprintf("blockchain/%s", createdBlockchainID))
+	t.NoError(err)
+	t.Equal(true, activatedBlockchain.Active)
+
+	/* ERROR - Create Blockchain (duplicate record) -> POST /blockchain */
+	_, err = post[repository.Blockchain]("blockchain", []byte(blockchainJSON))
+	t.Error(err)
```
Comment (pablocampogo) [inline tests/e2e_test.go:120]: We should compare the exact error we are expecting in all these tests, to avoid fake positives

**Review by pablocampogo — CHANGES_REQUESTED** [review]
As I said in a comment, please change all `t.Error` to exact assertions to the expected error

File: tests/e2e_test.go (hunk 1 above)
Comment (commoddity) [reply tests/e2e_test.go]: OK, updated.

**Review by pablocampogo — APPROVED** [review]
LGTM

**Pull request merged**